translate/helpers/jack_py_helpers.py

```diff
@@ -0,0 +1,66 @@
+import sys
```

Comment:
Are these helpers still experimental? If not, IMO this file should be called py_helpers.py to ensure that everyone reuses each other's code and don't reimplement things.